### DISCUSSION TITLE
Are GPU delegates enabled in this setup?

### DISCUSSION THREAD

**Discussion by jeremiahrose** [coqui-ai/STT#1979] [Q&A]
I already asked a similar question here https://github.com/coqui-ai/STT/discussions/1925

However, I'm still not 100% sure that the GPU delegates are being included in my custom build of Coqui. What is an easy way to find out for sure?

I'm building libstt.so with the following command. (The build is done inside Yocto / Openembedded so please just ignore all the custom crosstool_top stuff):
```
bazel build \
	    --workspace_status_command="bash native_client/bazel_workspace_status_cmd.sh" \
            --verbose_explanations \
	    -c opt \
	    --config=noaws \
	    --config=nogcp \
	    --config=nohdfs \
	    --config=nonccl \
            --cpu="${BAZEL_TARGET_CPU}" \
	    --crosstool_top=@local_config_yocto_compiler//:toolchain \
            --host_crosstool_top=@bazel_tools//tools/cpp:toolchain \
            --subcommands --explain=${T}/explain.log \
	    //native_client:libstt.so
```

I notice the docs say that experimental GPU delegate support is enabled for Android builds:

> TensorFlow Lite supports Delegate API to offload some computation from the main CPU. Please refer to TensorFlow’s documentation for details.
> 
> To ease with experimentations, we have enabled some of those delegations on our Android builds: * GPU, to leverage OpenGL capabilities * NNAPI, the Android API to leverage GPU / DSP / NPU * Hexagon, the Qualcomm-specific DSP

My build doesn't use the Android target however. Will GPU delegates be included in my build above?

Thanks!

**Comment by reuben** [comment]
They're enabled by default from our side, but if the Android configuration is not enabled they'll likely be disabled on the TensorFlow build side.
